hw/ip/kmac/pre_dv/kmac_reduced_tb/kmac_reduced_tb.core

```diff
@@ -0,0 +1,51 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv_verilator:kmac_reduced_tb"
+description: "KMAC Reduced Verilator TB"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:ip:kmac
+    files:
+      - rtl/kmac_reduced_tb.sv
+    file_type: systemVerilogSource
+
+  files_dv_verilator:
+    depend:
+      - lowrisc:dv_verilator:simutil_verilator
+
+    files:
+      - cpp/kmac_reduced_tb.cc
+    file_type: cppSource
+
+targets:
+  default:
+    default_tool: verilator
+    filesets:
+      - files_rtl
+      - files_dv_verilator
+    toplevel: kmac_reduced_tb
+    tools:
+      verilator:
+        mode: cc
+        verilator_options:
+# Disabling tracing reduces compile times by multiple times, but doesn't have a
+# huge influence on runtime performance. (Based on early observations.)
+          - '--trace'
+          - '--trace-fst' # this requires -DVM_TRACE_FMT_FST in CFLAGS below!
+          - '--trace-structs'
+          - '--trace-params'
+          - '--trace-max-array 1024'
+# compiler flags
+#
+# -O
+#   Optimization levels have a large impact on the runtime performance of the
+#   simulation model. -O2 and -O3 are pretty similar, -Os is slower than -O2/-O3
+          - '-CFLAGS "-std=c++11 -Wall -DVM_TRACE_FMT_FST -DTOPLEVEL_NAME=kmac_reduced_tb -g -O0"'
+          - '-LDFLAGS "-pthread -lutil -lelf"'
+          - "-Wall"
+          # XXX: Cleanup all warnings and remove this option
+          # (or make it more fine-grained at least)
```

Comment:
I realise that this is copy-pasted from other files, but is it actually needed here? If not, maybe we should drop it?

Reply:
We actually want that because for development and prototyping, the code you're experimenting with is not always lint clean. And that's fine because you're going back and forth and many iterations are typically needed. You only clean up lint once you're really done. If this option is removed, the first lint warning will cause Verilator to exit. I will change the comment to clarify this.

Reply:
Cool, sounds sensible. :-)